python/lsst/obs/sdss/selectSdssImages.py

```diff
@@ -180,6 +180,8 @@ def getColumnNames():
             "run rerun camcol field filter ra1 dec1 ra2 dec2 ra3 dec3 ra4 dec4".split() +
             "strip psfWidth sky airmass quality isblacklisted".split()
         )
+    def __hash__(self):
+        return id(self)
```

Comment:
Might be worth a one-line docstring along the lines of "necessary for hashing this object in python3." or somesuch.